atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -373,7 +374,7 @@ public Void execute(Transaction t) {
      */
     private boolean checkAndRepairTableDrop() {
         try {
-            Set<TableReference> tables = kvs.getAllTableNames();
+            Set<String> tables = kvs.getAllTableNames().stream().map(tableRef -> tableRef.getQualifiedName()).collect(Collectors.toSet());
```

Comment:
Could inline this into use site and use anyMatch, would also avoid looking up all tables for no progress result